src/components/pages/Page404.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+import './Page404.scss';
```

Comment:
Usually, this would be a good place for the import, in React.. But in this project should go in this file referenced/linked below, it's mostly because of the Rollup config and to allow for predefined Sass variables, mixins, functions, etc.. It also helps prevent style leakage, as well as a few other reasons that help make the project extensible.  

https://github.com/dragon3h/components/blob/master/src/assets/scss/styles.module.scss